packages/api2/src/periods/periods.controller.ts

```diff
@@ -121,4 +129,53 @@ export class PeriodsController {
   ): Promise<Praise[]> {
     return this.periodsService.praise(id);
   }
+
+  @Get(':id/verifyQuantifierPoolSize')
+  @ApiOperation({ summary: 'Verify quantifier pool size' })
+  @ApiResponse({
+    status: 200,
+    description: 'Period',
+    type: Period,
```

Comment:
Make sure all `@ApiResponse` types are correct. This one should be `VerifyQuantifierPoolSizeDto`.